meta/README.md

```diff
@@ -0,0 +1,37 @@
+# NuGet Proposal Process
+
+## How do I create a proposal?
+
+- Fork or clone https://github.com/NuGet/Home
+- Copy `meta/template.md` into `proposed/<year>/your-proposal-name.md`
+- Fill in the template with your proposal.
+- Submit a PR to the `NuGet/Home` repo
+- (Optional) Create a new discussion to talk about it https://github.com/NuGet/Home/discussions/new
```

Comment:
What about proposals for problems that have existing issues?

Reply:
The template should still have the issue link in it. I think it's the same process?

Reply:
good point. I guess I'm still not convinced we should tell people to use disc when there's so many of the topics they'd be interested in are tracked in issues.

Reply:
should we say something like "create a new discussion or start the conversation in an existing issue"?

Reply:
Yeah, I think that's fine.